### ISSUE TITLE
light build highlight.js version

### ISSUE THREAD

**Issue by forresto** [conorhastings/react-syntax-highlighter#68]
I really appreciate this project, and the [light build](https://www.npmjs.com/package/react-syntax-highlighter#light-build) directions. 

With
``` js
import js from 'highlight.js/lib/languages/javascript';
```
though I'm ending up with different versions of highlight.js in my project's node_modules vs react-syntax-highlighter's node_modules. It works, but I wonder if you could make a script similar to [build-stylesheets.js](https://github.com/conorhastings/react-syntax-highlighter/blob/master/lib/build-stylesheets.js) for the languages, so that the import could look more like:

``` js
import js from 'react-syntax-highlighter/dist/languages/javascript';
```

& maybe not need the `registerLanguage('javascript', js);`. I'd be happy to give it a shot if that sounds good to you.

**Comment by conorhastings** [comment]
@forresto yeah that sounds like a solid idea, not sure when i would have time to work on it but if you want to give it a shot, go for it.

**Comment by conorhastings** [comment]
closed via #69 and published in `v5.6.0`, thanks!
